src/configuration/routes/cache.md

```diff
@@ -23,58 +42,80 @@ https://{default}/:
 
-In this example, requests will be cached based on the URI, the `Accept` header, `Accept-Language` header, and `X-Language-Locale` header; Any response that lacks a `Cache-Control` header will be cached for 60 seconds; and the presence of any cookie in the response will disable caching of that response.
+## Cache configuration properties
 
-## Cache Per Route
+### `enabled`
+
+Turns the cache on or off for a route.
 
-If you need fine-grained caching, you can set up caching rules for
-several routes separately.
+**Type:** Boolean
 
-Here is an example:
+**Values**
+* `true`: enable the cache for this route
+* `false`: disable the cache for this route [default]
+
+### `headers`
+
+Adds specific header fields to the cache key, enabling caching of separate responses for those headers.
+
+For example, if the `headers` key is the following, Platform.sh will cache a different response for each value of the `Accept` HTTP request header:
 
 ```yaml
-https://{default}/:
-  type: upstream
-  upstream: app:http
-  cache:
-    enabled: true
+cache:
+  enabled: true
+  headers: ["Accept"]
+```
 
-https://{default}/foo/:
-  type: upstream
-  upstream: app:http
-  cache:
-    enabled: false
+**Type:** List
 
-https://{default}/foo/bar/:
-  type: upstream
-  upstream: app:http
-  cache:
-    enabled: true
+> See [Header behaviours](#Header_behaviours) for information on how specific headers affect the request.
+
+### `cookies`
+
+Defines which cookie names to include values for in the cache key.
+
+For example:
+
+```yaml
+cache:
+  enabled: true
+  cookies: ["foo"]
 ```
+The cache key will depend on the value of the `foo` cookie in the request.
 
-With this configuration, the following routes are cached:
+**Type:** List
 
--   `https://{default}/`
--   `https://{default}/foo/bar/`
--   `https://{default}/foo/bar/baz/`
+**Values:**
+* `*`: any request with a cookie will bypass the cache [default]
+* _user defined_: A valid cookie name
 
-And the following routes are **not** cached:
+> **Wildcards are not supported**
+> You can not use wildcards in the cookie name. Either use a precise cookie
+> name, or match all cookies with a `"*"`. `"SESS*"` or `"~SESS"` are currently
+> not valid values.
 
--   `https://{default}/foo/`
--   `https://{default}/foo/baz/`
+### `default_ttl`
 
-> **note**
-> Regular expressions in routes are **not** supported.
+Defines the default time-to-live for the cache in seconds. The cache duration is decided based on the `Cache-Control` response header value. If no `Cache-Control` header is in the response, then the value of `default_ttl` key (`0`) is used.
+
+The `default_ttl` only applies to **non-static responses**. 
 
-## Cache key
+**Type:** integer
 
-To decide how to cache a response, Platform.sh will build a cache key
-depending on several factors and store the response associated with this
-key. When a request comes with the same cache key, the cached response will
-be reused.
+**Values:**
+* `0`: Cache indefinitely [default]
```

Comment:
A max-age of 0 is is don't cache, not cache indefinitely.  Unless we special case reverse that, too, which we really shouldn't. :smile: